testing/camunda-process-test-spring/src/main/java/io/camunda/process/test/api/CamundaProcessTestExecutionListener.java

```diff
@@ -57,8 +59,10 @@
  * <ul>
  *   <li>Publish a {@link CamundaClientClosingEvent}
  *   <li>Close created {@link CamundaClient}s
- *   <li>Stop the runtime
+ *   <li>Purge the Zeebe cluster
```

Comment:
:wrench: Align JavaDoc with JUnit extension.

```suggestion
 *   <li>Purge the runtime (i.e. delete all data)
```